components/version_info/android/BUILD.gn

```diff
@@ -23,7 +23,17 @@ process_version("generate_brave_version_constants") {
     "BRAVE_BROWSER_VERSION=\"$brave_version\"",
     "-e",
     "BRAVE_CHROMIUM_VERSION=\"$chrome_version_string\"",
-    "-e",
-    "BRAVE_CHANNEL=str.upper('$brave_channel')",
   ]
+
+  if (brave_channel == "") {
```

Comment:
I think it should be something like this:
`if (is_official_build && brave_channel == "") {`

according to `//brave/common/brave_channel_info.cc`

Reply:
done